README.md

```diff
@@ -1,56 +1,22 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# oop-school-library
 
-# Project Name
-
-> Description the project.
+> This is a Ruby project for school library
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- Ruby
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+This repository includes files with plain Ruby code
```

Comment:
- [ ] your README looks good but It is missing this important part of the `Getting started ` section. Normally, this section should contain procedures to clone and run your project locally. 😃  As you understand, It is important for your fellow developers. Please add more content to this section as I mentioned above 😃 👍